### ISSUE TITLE
Prepopulated Fields not Working on NestedInlines

### ISSUE THREAD

**Issue by farizidfl** [theatlantic/django-nested-admin#43]
It seems that both _prepopulated_fields_ attriibute and _get_prepopulated_fields()_ hook method do not trigger the JavaScript for it.

```
class CalculationFieldInline(nested_admin.NestedTabularInline):
    model = CalculationField
    prepopulated_fields = {'keyword': ('name',)}

    def get_prepopulated_fields(self, requests, obj):
        return {'keyword': ('name',)}
```
It works if I simply replace the NestedTabularInline to regular TabularInline in the code above.

**Comment by fdintino** [comment]
I'm guessing you are using Grappelli? There is a known bug with django-grappelli, django-nested-admin, and `prepopulated_fields` (there is a unit test for it that is currently skipped because it fails, [see here](https://github.com/theatlantic/django-nested-admin/blob/bd2896c062a7755017e536c2970bfe934ef302c7/nested_admin/tests/admin_widgets/tests.py#L101-L108)). If it is this bug then I can take a look at fixing it... when I wrote those tests I didn't have time to address the underlying issue.

**Comment by farizidfl** [comment]
Ah yes, I am using Grappelli and the bug does disappear when I remove Grappelli. That would be much appreciated!

**Comment by fdintino** [comment]
Fixed with the 3.0.4 release
